EIPS/eip-7378.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 7378
+title: Add time-weighted averaging to the base fee
+description: Using geometric weights to average past block sizes into consideration
+author: Guy Goren (@guy-goren) <guy.nahsholim@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/add-time-weighted-averaging-to-the-base-fee-mechanism/15142
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-07-22
+---
+
+## Abstract
+
+This EIP proposes a new formula for the base fee, derived from [EIP-1559](./eip-1559.md). The existing base fee formula,
```

Comment:
```suggestion
This EIP proposes a new formula to update the base fee, derived from [EIP-1559](./eip-1559.md). The existing base fee is updated as follows,
```

Reply:
also wouldn't the cost for empty blicj be based off on full block size and not the target block size (so cost increases 2x)